Patient dashboard

AndyMG1999 opened the pull request:
Patient notes no longer works but, dashboard has been successfully switched to recyclerview

AndyMG1999 committed:
Dashboard Initial Design

XML and Java File Created
WARNING: Trying to fix errors with crashing app

AndyMG1999 committed:
Update patient_dashboard.xml

AndyMG1999 committed:
Added Java File For Dashboard

AndyMG1999 committed:
Notes pages

added xml and java file for patient notes page

AndyMG1999 committed:
added google services json

AndyMG1999 committed:
deleted google-services (sorry :P)

AndyMG1999 committed:
Replaced Scrollview with Recyclerview

-Switched and set up recyclerview for patient dashboard
-Set up recyclerview for patient notes

AndyMG1999 committed:
Tried to switch notes to recyclerview :(

chelseaL01 commented on client/app/src/main/res/layout/patient_notes.xml:
```diff
@@ -0,0 +1,44 @@
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.061" />
+
+    <androidx.recyclerview.widget.RecyclerView
```
Add the ID "patientNotesRecyclerView" to this RecyclerView

chelseaL01 replied:
Add the ID "patientNotesRecyclerView" to this RecyclerView.

chelseaL01 commented on client/app/src/main/java/com/example/protego/PatientNotesActivity.java:
```diff
@@ -0,0 +1,66 @@
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.patient_notes);
+
+        RecyclerView recyclerView = findViewById(R.id.patientNotesRecyclerView);
```
This activity crashes here because the ID "patientNotesRecyclerView" is set to the layout instead of the recycler view.  Minor changes in the XML file will resolve the issue. Please refer to my other comments.

chelseaL01 commented on client/app/src/main/res/layout/patient_notes.xml:
```diff
@@ -0,0 +1,44 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/patientNotesRecyclerView"
```
A solution is to change to ID of this layout.

chelseaL01 reviewed (commented):
Potential solution to the patient's notes Recycler View crash

AndyMG1999 committed:
Fixed my very silly error

I named my constaint layout what my recyclerview was supposed to be named... fixed it! (Thanks Chelsea)

chelseaL01 committed:
Add navigation bar to patient activities + Add patient name to navbar

AndyMG1999 committed:
Added QR Code Page

Added QR Code page and am now trying to code in functionality

AndyMG1999 committed:
Added navigation to QR Code

QR Code for patient can be reached from the dashboard now

tachyon161 committed:
add notification menu design to dashboard

tachyon161 committed:
add bottom sheet for notifications

tachyon161 committed:
add navigation to notifications

tachyon161 committed:
add swipe up feature to menu

tachyon161 committed:
Merge branch 'main' into patient-dashboard

tachyon161 committed:
fix bugs

chelseaL01 committed:
Add connection to Patient Dashboard from Login Activity

tachyon161 committed:
Merge branch 'main' into patient-dashboard

tachyon161 committed:
add userType check to login (commented out for now)
